pyproject.toml

```diff
@@ -286,3 +285,349 @@ lint.ignore = [
 "jwst/ramp_fitting/tests/*" = [
     "E402", # module-level import not at top of file
 ]
+
+[tool.towncrier]
+filename = "CHANGES.rst"
+directory = "changes"
+package = "jwst"
+title_format = "{version} ({project_date})"
+ignore = [".gitkeep"]
+wrap = true
+issue_format = "`#{issue} <https://github.com/spacetelescope/jwst/issues/{issue}>`_"
+
+[[tool.towncrier.type]]
+directory = "general"
+name = "General"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "docs"
+name = "Documentation"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "stpipe"
+name = '``stpipe``'
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "datamodels"
+name = "Data Models"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "associations"
+name = "Associations"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "scripts"
+name = "Scripts"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "fits_generator"
+name = "``fits_generator``"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "set_telescope_pointing"
+name = '``set_telescope_pointing``'
+showcontent = true
+
```

Comment:
Suggest adding a `pipeline` entry for PRs modifying the files in `jwst/pipeline`.

Reply:
```suggestion

# pipeline

[[tool.towncrier.type]]
directory = "pipeline"
name = 'Pipeline'
showcontent = true
```